core/contracts/common/AccessControl.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.5.0;
+
+import "./interfaces/AccessControlInterface.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+
+// Sourced from proposed OpenZeppelin implementation of ERC20Mintable with v3 contracts
+// - description: https://forum.openzeppelin.com/t/redesigning-access-control-for-the-openzeppelin-contracts/2177/24
+// - source code: https://gist.github.com/nventuro/ac42bd936f678ebf55c4c0c4a4ee4c72
+
+/** @dev Each role has an associated admin role, and a role can only be granted or revoked by accounts that have it’s admin role
+ */
+contract AccessControl is IAccessControl {
```

Comment:
I think we should just use our own MultiRole contract rather than taking open zeppelin's draft AccessControl solution.

Reply:
agreed